src/main/kotlin/com/hypto/iam/server/di/Modules.kt

```diff
@@ -150,6 +152,8 @@ val applicationModule = module {
         }.connectionPool(ConnectionPool(MAX_IDLE_CONNECTIONS, KEEP_ALIVE_DURATION, TimeUnit.MINUTES))
     }
     single(named("AuthProvider")) { get<OkHttpClient.Builder>().build() }
+    single { AuthProviderRegistry }
+    single(null, true) { GoogleAuthProvider }
```

Comment:
Add a TODO saying that we'll have to fix auto registration (without adding it to Modules.kt) using Koin `@ComponentScan` and `@Module` annotations